app/src/main/java/com/example/drawingactivity/DrawActivity.kt

```diff
@@ -58,12 +61,12 @@ class DrawActivity : AppCompatActivity() {
         }
 
         viewModel.onUpdate = {
-            if (it.layerList.isEmpty()) {
-                viewModel.addLayer(this.getString(R.string.layerHint, 1))
-            }
             colorSelector.circlePrimaryColor = it.primaryColor
             drawableArea.pathPaint.color = it.primaryColor
             drawableArea.userDrawnBitmap = it.currentBitmap
+            if (it.layerList.isEmpty()) {
```

Comment:
I don't think we should do this, it seems a bit hacky. Don't let them delete their last layer, as in throw an error. You don't need to do that in this ticket though, but this if block seems like a hacky way to fix a symptom of a different problem

Reply:
Added logic to check the list size before deleting to prevent cases where users can delete the first layer in general, I will likely fix how it initializes in a future ticket since I need to make sure it grabs the correct localized string when it first is made. Once I finish this ticket I may take a crack at it in another PR with another ticket.